src/rcx/test/generate_pattern.ok

```diff
@@ -2,6 +2,7 @@
 [INFO ODB-0223]     Created 13 technology layers
 [INFO ODB-0224]     Created 25 technology vias
 [INFO ODB-0226] Finished LEF file:  sky130hs/sky130hs.tlef
+[INFO RCX-0437] RECT li1 ( 9910 85 ) ( 10080 16915 )  jids= ( 1 2 )
```

Comment:
This seems spurious as well.

Reply:
The test seems to fail at master without including this.

Reply:
The master branch is green so I don't see how this can be required unless it somehow relates to your changes.